packages/network/src/logic/Node.ts

```diff
@@ -31,8 +32,6 @@ export enum Event {
     DUPLICATE_MESSAGE_RECEIVED = 'streamr:node:duplicate-message-received',
     NODE_SUBSCRIBED = 'streamr:node:subscribed-successfully',
     NODE_UNSUBSCRIBED = 'streamr:node:node-unsubscribed',
-    PROXY_CONNECTION_ACCEPTED = 'streamr:node:proxy-connection-accepted',
-    PROXY_CONNECTION_REJECTED = 'streamr:node:proxy-connection-rejected',
     ONE_WAY_CONNECTION_CLOSED = 'stream:node-one-way-connection-closed',
```

Comment:
I would prefer to keep the code as is. There is really no way in the current network to know if an interaction between network participants has been fulfilled successfully outside of emitting events upon completion. `waitForCondition` can be used to check that a node has reached a desired state but it will not ensure that an operation has been successfully completed.